lib/twitter_cldr/resources/download.rb

```diff
@@ -6,8 +6,13 @@
 module TwitterCldr
   module Resources
 
-    CLDR_URL = 'http://unicode.org/Public/cldr/26/core.zip'
-    ICU4J_URL = 'http://download.icu-project.org/files/icu4j/54.1/icu4j-54_1.jar'
+    CLDR_VERSION    = '26'
```

Comment:
We have `CLDR_VERSION = '26.0'` in `lib/twitter_cldr.rb`. Can we combine these two somehow? I'd also suggest moving version constants for Unicode and ICU into `lib/twitter_cldr.rb` too.